xml/ftp_banners.xml

```diff
@@ -463,6 +505,7 @@ more text</example>
     <example service.version="0.9.13a beta">FileZilla Server version 0.9.13a beta</example>
     <example service.version="0.9.54 beta">FileZilla Server 0.9.54 beta</example>
     <example service.version="0.9.33 beta">FileZilla Server v0.9.33 beta</example>
+    <param pos="0" name="service.vendor" value="Filezilla-Project"/>
```

Comment:
There is no NIST issued CPE value for the Filezilla FTP server.  This change just aligns with vendor value with the vendor in the existing NIST CPEs for the client.